src/test/java/com/integrivideo/Data.java

```diff
@@ -21,6 +21,9 @@
     public final static String INSTALLED_CHAT_WITH_DOMAIN = "http://dmitryrak11.000webhostapp.com/COMPONENT_CHAT_WITH_DOMAIN/";
     public final static String LOGIN_PAGE_URL = BASE_URL + "login";
     public final static String NAME = "Guest";
```

Comment:
I don't like idea of hardcoding strings. How about hardcoding them inside pageobject and assertions just by the prices?